models/stdp_symmetric_connection.h

```diff
@@ -0,0 +1,300 @@
+/*
+ *  stdp_connection.h
```

Comment:
In the copyright notice, the correct file name should be stated: `stdp_symmetric_connection.h`. This is why `regressiontests/ticket-659-copyright.py` fails.